### PR TITLE
Update io.github.mideb.digitales_register runtime to 46

### PR DESCRIPTION
**Pull request by yakushabb**
- Update io.github.mideb.digitales_register runtime to 46
- Fix appdata papercuts

### CODE REVIEW

**Comment by ghost** [comment]
@miDeb I've updated io.github.mideb.digitales_register runtime to 46 since runtime version 43 has reached end-of-life.

**Comment by ghost** [comment]
@bbhtt Ready for review.

**Comment by bbhtt** [comment]
Can you try 47 instead? 46 has only 5 months left until EOL.

**Comment by ghost** [comment]
There is an error preventing the app from building against runtime 47.

**Comment by bbhtt** [comment]
Hm ok

bot, build

File: io.github.mideb.digitales_register.yaml [hunk 1]
```diff
@@ -40,6 +40,7 @@ modules:
       - install -Dm644 digitales_register/digitales_register.desktop /app/share/applications/io.github.mideb.digitales_register.desktop
       - install -Dm644 digitales_register/digitales_register.metainfo.xml /app/share/appdata/io.github.mideb.digitales_register.metainfo.xml
       - install -Dm644 digitales_register/icon.png /app/share/icons/hicolor/128x128/apps/io.github.mideb.digitales_register.png
+      - sed -i 's|https://schule.digitalesregister.it|schule.digitalesregister.it|' /app/share/appdata/io.github.mideb.digitales_register.metainfo.xml
```
Comment (bbhtt) [inline io.github.mideb.digitales_register.yaml]: any reason you are changing this? we allow https links in descriptions even though appstream upstream does not.

Comment (ghost) [reply io.github.mideb.digitales_register.yaml]: To pass appstream validation.

**Pull request merged**